classes/StandardControlsRegistry.php

```diff
@@ -106,13 +106,31 @@ protected function registerCheckboxControl()
 
     protected function registerSwitchControl()
     {
-        $this->controlLibrary->registerControl(
-            'switch',
+        $properties = [
+            'on' => [
+                'title' => Lang::get('rainlab.builder::lang.form.property_switch_label_on'),
+                'description' => Lang::get('rainlab.builder::lang.form.property_switch_label_on_description'),
+                'type' => 'string',
+                'ignoreIfEmpty' => true,
+                'sortOrder' => 70
+            ],
+            'off' => [
+                'title' => Lang::get('rainlab.builder::lang.form.property_switch_label_off'),
+                'description' => Lang::get('rainlab.builder::lang.form.property_switch_label_off_description'),
+                'type' => 'string',
+                'ignoreIfEmpty' => true,
+                'sortOrder' => 71
+            ],
+        ];
+
+        $properties = array_merge($properties, $this->getCheckboxTypeProperties());
+
+        $this->controlLibrary->registerControl('switch',
```

Comment:
@webbati Please put the `'switch'` argument on a new line, as this has pinged our code quality checks here:
https://travis-ci.org/rainlab/builder-plugin/jobs/609788915#L3619

Reply:
@bennothommo, I did it